.eslintignore

```diff
@@ -65,22 +65,20 @@ npm/cypress-schematic/src/**/*.js
 # community templates we test against, no need to lint
 system-tests/projects/create-react-app-configured/**/*
 system-tests/projects/create-react-app-unconfigured/**/*
+
 system-tests/projects/vueclivue2-unconfigured/**/*
 system-tests/projects/vueclivue2-configured/**/*
+
 system-tests/projects/vueclivue3-unconfigured/**/*
 system-tests/projects/vueclivue3-configured/**/*
+
 system-tests/projects/vue3-vite-ts-unconfigured/**/*
 system-tests/projects/vue3-vite-ts-configured/**/*
+
 system-tests/projects/nextjs-unconfigured/**/*
 system-tests/projects/nextjs-configured/**/*
+
 system-tests/projects/nuxtjs-vue2-unconfigured/**/*
 system-tests/projects/nuxtjs-vue2-configured/**/*
-system-tests/projects/create-react-app-configured/**/*.js
-system-tests/projects/create-react-app-unconfigured/**/*.js
-system-tests/projects/vueclivue3-unconfigured/**/*
-system-tests/projects/vueclivue3-configured/**/*
-system-tests/projects/nextjs-configured/**/*
-system-tests/projects/nextjs-configured/**/*
-system-tests/projects/nuxtjs-vue2-unconfigured/**/*
-system-tests/projects/nuxtjs-vue2-configured/**/*
+
```

Comment:
There are mostly community templates (think CRA, Vite, Next...) so I'd don't want to lint them - just ignoring them for now.